forms/htmleditor/TinyMCEConfig.php

```diff
@@ -34,6 +34,7 @@ class TinyMCEConfig extends HtmlEditorConfig {
 		'remove_script_host' => true,
 		'convert_urls' => false, // Prevent site-root images being rewritten to base relative
 		'menubar' => false,
+		'language' => 'en',
```

Comment:
Set a default here... maybe some logic could check if the current i18n locale is also a valid tinymce locale? Potential future improvement.